mypyc/irbuild/expression.py

```diff
@@ -266,13 +266,20 @@ def transform_super_expr(builder: IRBuilder, o: SuperExpr) -> Value:
 
 
 def transform_call_expr(builder: IRBuilder, expr: CallExpr) -> Value:
+    callee = expr.callee
     if isinstance(expr.analyzed, CastExpr):
         return translate_cast_expr(builder, expr.analyzed)
     elif isinstance(expr.analyzed, AssertTypeExpr):
         # Compile to a no-op.
         return builder.accept(expr.analyzed.expr)
+    elif (
+        isinstance(callee, (NameExpr, MemberExpr))
+        and isinstance(callee.node, TypeInfo)
+        and callee.node.is_newtype
+    ):
+        # Calls to a NewType type is a no-op at runtime.
```

Comment:
```suggestion
        # A call to a NewType type is a no-op at runtime.
```
grammar nit

Reply:
I'll commit this after CI finishes. I rely on CI to fully test my patches since my laptop is too slow to run the mypyc tests locally.

Reply:
@ichard26 I may look into speeding up mypyc tests on macOS using Docker. Are you using a mac by any chance?

Reply:
I'm using a Linux laptop so that won't be helpful. However, I've gotten around to setting up a Codespaces instance. A 8-core VM runs the whole mypyc test suite in ~2 minutes which is more than fast enough!

![image](https://user-images.githubusercontent.com/63936253/211074340-7c0655c9-f22e-4062-a261-c808753b4ec8.png)